# HG changeset patch
# User Ben Abbott <bpabbott@mac.com>
# Date 1234397682 18000
#      Wed Feb 11 19:14:42 2009 -0500
# Node ID 0899370cce9f269fed48a180e48682a19fd7c09c
# Parent  3cedb606145dea4884c66479bf7e05b5ef9cc89a
Include FTGLTextureFont.h when FTGL_UPPER_CASE is not defined.

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,13 @@
+2009-02-11 Ben Abbott <bpabbott@mac.com>
+
+	* configure.in: Include FTGLTextureFont.h when FTGL_UPPER_CASE is
+	not defined.
+
 2009-02-11  John W. Eaton  <jwe@octave.org>
 
 	* configure.in (octetcdir): New default value.
 	* Makeconf.in (octetcdir): New variable.
 	(do-subst-default-vals): Substitute octetcdir.
 	* octMakefile.in (install, uninstall): Use $(octetcdir) for NEWS.
 	(DIRS_TO_MAKE): Add $(octetcdir) to the list.
 
diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -703,27 +703,27 @@ if test -n "$OPENGL_LIBS"; then
 	  ], has_ftgl_h=no)])
       if test "$has_ftgl_h" = yes; then
 	AC_MSG_CHECKING([for FTGLTextureFont in -lftgl])
 	LIBS="$LIBS -lftgl"
         AC_TRY_LINK([
 #ifdef HAVE_FTGL_FTGL_H
 #ifdef HAVE_FTGL_UPPERCASE
 #include <FTGL/FTGL.h>
-#include <FTGL/FTGLTextureFont.h>
 #else
 #include <FTGL/ftgl.h>
 #endif
+#include <FTGL/FTGLTextureFont.h>
 #elif HAVE_FTGL_H
 #ifdef HAVE_FTGL_UPPERCASE
 #include <FTGL.h>
-#include <FTGLTextureFont.h>
 #else
 #include <ftgl.h>
 #endif
+#include <FTGLTextureFont.h>
 #endif], [
 FTGLTextureFont font("");], [
           OPENGL_LIBS="$OPENGL_LIBS $FT2_LIBS -lftgl"
 	  LIBS="$save_LIBS"
 	  CPPFLAGS="$save_CPPFLAGS"
           AC_DEFINE(HAVE_FTGL, 1, [Define to 1 if FTGL is present])
 	  AC_MSG_RESULT(yes)
 	  XTRA_CXXFLAGS="$XTRA_CXXFLAGS $FT2_CFLAGS"],[
